src/test/java/seedu/toluist/model/TodoListTest.java

```diff
@@ -64,6 +64,26 @@ public void constructor_withStorage_loadTasks() {
     }
 
     @Test
+    public void constructor_replaceCurrentTodoList() {
+        TodoList anotherTodoList = new TodoList();
+        assertEquals(anotherTodoList, TodoList.load());
+    }
+
+    @Test
+    public void save_replaceCurrentTodoList() {
+        TodoList anotherTodoList = new TodoList();
+        anotherTodoList.add(new Task("ask"));
+        TodoList emptyTodoList = new TodoList();
```

Comment:
its actually `used`. Look at the previous test case, constructitng an object is doing sth